.circleci/config.yml

```diff
@@ -40,12 +40,12 @@ aliases:
 executors:
   node-executor:
     docker:
-      - image: circleci/node:13
+      - image: circleci/node:latest
```

Comment:
Currently the `latest` tag is still resolved to `13.10`. After CircleCI updates their docker image, we will move to `14` automatically.